qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -197,20 +212,33 @@ def _score_layout(self, layout):
         to weight against higher connectivity qubits."""
         bits = layout.get_physical_bits()
         score = 0
-        if self.properties is None:
-            # Sum qubit degree for each qubit in chosen layout as really rough estimate of error
+        if self.target:
+            if "measure" in self.target:
+                for bit in bits:
+                    props = self.target["measure"].get((bit,))
+                    if props is None or props.error is None:
+                        score += (
+                            self.coupling_map.graph.out_degree(bit)
+                            + self.coupling_map.graph.in_degree(bit)
+                        ) / len(self.coupling_map.graph)
+                    else:
+                        score += props.error
+        else:
+            if self.properties is None:
+                # Sum qubit degree for each qubit in chosen layout as really rough estimate of error
+                for bit in bits:
+                    score += self.coupling_map.graph.out_degree(
+                        bit
+                    ) + self.coupling_map.graph.in_degree(bit)
+                return score
```

Comment:
This is pre-existing and doesn't need to be changed (nor should it, this close to release!), but it's weird to me that this path doesn't divide by the size of the graph, unlike other paths this approximation is used in.

Reply:
It's mostly a matter of relative values on the other qubits for the heuristic score that's returned. In the other paths when we have an error rate on some qubits that's always going to be < 1. So in the other paths we divide by the number of nodes in the absence of an error rate to make the value < 1 to hopefully not score super unfavorably towards qubits that could be potentially good and just happen to be missing an error rate in the backend but are still defined in the connectivity graph.

In the case of this path when we have no error rates it doesn't really matter because all the other scores are going to be in the same situation so we want to prefer to use a node with lower degree. We could divide but we don't have to as it wouldn't change the relative score compared to the other qubits.